webapp/pom.xml

```diff
@@ -130,6 +130,12 @@
         <dependency>
             <groupId>org.apache.atlas</groupId>
             <artifactId>atlas-notification</artifactId>
+            <exclusions>
```

Comment:
Do we need yammer's metric core as well?

Reply:
metrics-core from yammer is dependency coming from kafka dependency in notification which is compile dependency.
I checked that notifications are send correctly and there is no exception from notification/ kafka related exception.
so we can exclude this metrics-core from yammer.